File: ts/iq/tos~get~get_disclosure_stage_by_id.ts
/** <iq type="get" xmlns="tos">. Replies with one of: ClientSuccess, ClientError, ServerError. */
export function buildUserNoticeGetDisclosureStageByIds(id: number, t: number): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'tos', type: 'get' }, content: [
      { tag: 'get_disclosure_stage_by_id', attrs: { id: id, t: t } }, /* repeated */
    ] }
}

File: rust/iq/tos~get~get_disclosure_stage_by_id.rs
/// <iq type="get" xmlns="tos">. Replies with one of: ClientSuccess, ClientError, ServerError.
pub fn build_user_notice_get_disclosure_stage_by_ids(id: i64, t: i64) -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "tos")
        .attr("type", "get")
        .child(NodeBuilder::new("get_disclosure_stage_by_id").attr("id", id).attr("t", t).build()) // repeated
        .build()
}